frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -198,6 +217,278 @@ export class OutputArtifactLoader {
     };
   }
 
+  public static async getMlmdContext(kfpPodName: string): Promise<Context> {
+    if (kfpPodName.split('-').length < 3) {
+      throw new Error('kfpPodName has fewer than 3 parts');
+    }
+
+    const pipelineName = kfpPodName
```

Comment:
Can you document what kfpPodName should look like, and then describe what pipelineName, runID and contextName will be as a result?

Reply:
Done. Also renamed kfpPodName to ArgoPodName, as within KFP code, naming should be mor explicit.